install/config.py

```diff
@@ -16,3 +18,18 @@ def get(self, key):
             return self.config[key]
         else:
             return None
+    def findVenv(self):
+        os.system("whereis virtualenv > foundit.txt")
+        with open('foundit.txt', 'r') as fp:
+            found = fp.read()
+        os.system("rm foundit.txt")
+        found = re.search(r"(?P<venvpath>/.*/virtualenv)[\n\s]", found)
+        if found:
+            return found.group('venvpath')
+        else:
+            return None
```

Comment:
yeah uh

use a subprocess call to read the output. Also what is the regex for? Shouldn't the first line just give you the full path? Why do you need to seperate the path from `virtualenv`?

Reply:
The first line gives any and all paths it found the file at.
There's both preceding text, as well as another virtualenv file elsewhere that has appendations
`virtualenv: /usr/local/bin/virtualenv /usr/share/man/man1/virtualenv.1.gz`
^That, basically.

Reply:
Got the subprocess to work, I was missing `shell=True` I believe. I'll push that in a bit

Reply:
Okay I see, that was. I was thinking this worked more like `which`, but yeah makes sense. Still partial to not using regex as much as possible. Maybe result.split()[1]? Regex group matching can have some interesting effects..

Reply:
It's not always guaranteed to be the 1st, or 2nd, or 0th index though, that's the thing.
I suppose an alternative would be to split, and use .endswith() or something.

I made it a really simple regex though... oh, it needs a question mark after the asterisk so it's not greedy, but then it's only grabbing the path before the file where virtualenv is preceeded by either a space, or a newline, with no other characters afterwards.
I'll put an 'end of line' in there just to be robust, but otherwise this should be as efficient and tidy as regex gets. It's a simple formula. And we're not exactly running it through extensive edge cases on people's computers either.